modules/ROOT/pages/monitoring-dashboards.adoc

```diff
@@ -215,15 +215,17 @@ See the <<Breakdown Tab,Breakdown tab>> for details about each of the components
 CPU::
 This metric is not available on Windows.
 +
-The total CPU usage of the server, as a percentage:
+The total CPU usage of the server:
 
 * CPU usage
```

Comment:
Should the case match the graph? ie, CPU Usage, Available Processors, Load Average?